src/query/expression/src/kernels/tests.rs

```diff
@@ -0,0 +1,263 @@
+// Copyright 2021 Datafuse Labs
```

Comment:
For `pub` functin tests, would be better to put to:
https://github.com/datafuselabs/databend/tree/main/src/query/expression/tests/it

Reply:
Thx, done.